apps/examples/log_dump/log_dump_main.c

```diff
@@ -57,9 +75,20 @@ int log_dump_main(int argc, char *argv[])
 			lldbg_noarg("%c", buf[i]);
 		}
 	}
+
+	if (STOP_LOGDUMP_SAVE(fd) < 0)
+	{
+		printf("Fail to stop log dump, errno %d\n", get_errno());
+		goto errout;
+	}
```

Comment:
What I understand is that the stop is added to read saved log. But this example doesn't. Could you confirm the purpose of new functions?

Reply:
Modified to stop before reading.